tsconfig.json

```diff
@@ -36,7 +36,8 @@
       "@global-images/*": ["src/assets/img/*"],
       "@eco-news-images/*": ["src/assets/img/icon/econews/*"],
       "@footer-images/*": ["src/assets/img/icon/footer/*"],
-      "@environment/*": ["src/environments/*"]
+      "@environment/*": ["src/environments/*"],
+      "@shared/*": ["src/app/component/shared/*"]
```

Comment:
Now the list of aliases is ordered:
![image](https://user-images.githubusercontent.com/6242270/89627928-342fb780-d8a4-11ea-83f6-29b9f63248db.png)
So, put your alias according to alphabetical order (by paths) .

Reply:
fixed